src/coreclr/utilcode/longfilepathwrappers.cpp

```diff
@@ -534,8 +534,8 @@ CopyFileExWrapper(
 
 //Implementation of LongFile Helpers
 const WCHAR LongFile::DirectorySeparatorChar = W('\\');
-const WCHAR LongFile::AltDirectorySeparatorChar = W('/');
 #ifdef HOST_WINDOWS
+const WCHAR LongFile::AltDirectorySeparatorChar = W('/');
```

Comment:
Why are we compiling any of this for Unix? Should this file be Windows-specific?

Reply:
This file contains also wrappers for functions that return string in a buffer and those wrappers get SString argument. But I will ifdef out the functions that are not this case and the LongFile class.